languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -2125,11 +2125,34 @@ USA
     <!-- ====================================================================== -->
     <category id="TYPOS" name="Possible Typo" type="misspelling">
         <rule id="CAN_MISSPELLING" name="misspelling can" default="temp_off">
+        <!-- TODO: work in progress, need to add AP to avoid 'Support the Ring Cam please' -->
```

Comment:
name=" *incorrect sentence* (*correct suggestion*)"

For example:
name="It cam (can) happen anytime"